src/cfnlint/__init__.py

```diff
@@ -197,30 +203,33 @@ def __len__(self):
 
     def extend(self, more):
         """Extend rules"""
-        self.rules.extend(more)
+        for rule in more:
+            if self.is_rule_valid(rule):
+                self.rules.append(rule)
 
     def __repr__(self):
         return '\n'.join([rule.verbose()
                           for rule in sorted(self.rules, key=lambda x: x.id)])
 
-    def resource_property(self, filename, cfn, ignore_checks, path, properties, resource_type, property_type):
+    def is_rule_valid(self, rule):
```

Comment:
`Valid` sound a bit weird. Can the naming be improved a bit? `is_rule_enabled` or something?